packages/website/src/components/Playground.module.css

```diff
@@ -43,6 +43,7 @@
   height: calc(100% - var(--ifm-navbar-height));
   top: var(--ifm-navbar-height);
   z-index: calc(var(--ifm-z-index-fixed) - 1);
+  background: var(--playground-main-color);
```

Comment:
Is this needed? The only change I can see is in dark mode, and adding this background in makes the tabs less distinct from the background.

```suggestion
```

Reply:
yes, if there is delay while updating layout of editor, we can see 2 distinct backgrounds,
i'm going to update tabs to have correct bg

Reply:
i'm going to recheck if it looks ok without background changes :)

Reply:
ok i reverted color changes, but those colors where afecting only 

![image](https://user-images.githubusercontent.com/625469/220843112-17493169-fd55-4bbb-82fd-c58558a8762f.png) ![image](https://user-images.githubusercontent.com/625469/220843138-c62f3d30-db15-4436-abf6-f45781977072.png)

and bacgkground that is under editor while we are loading it or resizing

![image](https://user-images.githubusercontent.com/625469/220843232-665fdbec-1859-491a-b643-330e3a4d0441.png)
![image](https://user-images.githubusercontent.com/625469/220843286-9f1ce3bc-6611-4cf2-ad31-6500d4f3bf32.png)